samples/display/lvgl/prj.conf

```diff
@@ -1,4 +1,6 @@
 CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_LOG_STRDUP_BUF_COUNT=16
```

Comment:
Looks like settings `CONFIG_LOG_STRDUP_BUF_COUNT` gives a warning on `native_posix`

Reply:
This is because native_posix does not use log_strdup. In fact, the stack size does not need to be changed for posix. It seems to be more platform specific than board specific, so I'm not sure how they should be handled. One option would be to change LVGL log settings (e.g. error or off) so that default `CONFIG_LOG_STRDUP_BUF_COUNT` will work in most cases. In any case, it would probably be good to document this part.